app/utils/general/index.ts

```diff
@@ -51,3 +52,5 @@ export const sortByNewest = (a: SortByCreatAt, b: SortByCreatAt) => {
 
     return 1;
 };
+
+export const isBetaApp = DeviceInfo.getBundleId && DeviceInfo.getBundleId().includes('rnbeta');
```

Comment:
why the check for `DeviceInfo.getBundleId`?

Reply:
This constant for `isBetaApp` can also be used in `init/push_notifications.ts#L251`

Reply:
corrected

Reply:
I don't see the correction